src/logic/calculate.js

```diff
@@ -0,0 +1,129 @@
+import operate from './operate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
+}
+
+/**
+ * Given a button name and a calculator data object, return an updated
+ * calculator data object.
+ *
+ * Calculator data object contains:
+ *   total:s      the running total
+ *   next:String       the next number to be operated on with the total
+ *   operation:String  +, -, etc.
+ */
```

Comment:
- [ ] It will be nice to clean up your code base. Please consider removing all commented code and comments from your code base.